doc/adding-new-user-permissions.md

```diff
@@ -1,3 +1,5 @@
+## **This permission model is deprecated** _(October 2020)_
```

Comment:
I think you don't need to bold it if it's an H2 😅

Reply:
😎